kinbot/homolytic_scissions.py

```diff
@@ -71,14 +71,25 @@ def find_homolytic_scissions(self):
                         # get the elements of the current bond under consideration, and put them
                         # in alphabetical order
                         bond_elements = sorted([self.species.atom[i], self.species.atom[j]])
+                        b_e = []
```

Comment:
Code updated to make sure that user input homolytic scission elements are sorted and listed in the same was as the `bond_elements` variable. 

For example
user input = `OC` --> needs to be sorted to be listed as `CO` in the same way that  bond_elements = `CO`. This check helps ensure that the user input matches the `bond_elements` variable when kinbot checks. 

Without this code, a user input of `OC` will be read as `OC != CO` even though `CO` and `OC` should be considered as the same type of bond being targeted for homolytic scission reactions.